src/devtoolsPanel.ts

```diff
@@ -298,31 +298,44 @@ export class DevToolsPanel {
     }
 
     private getHtmlForWebview() {
-        const htmlPath = vscode.Uri.file(path.join(this.extensionPath, 'out/tools/front_end', 'inspector.html'));
-        const htmlUri = this.panel.webview.asWebviewUri(htmlPath);
+        // stringsUri and inspectorUri are the files that used to be loaded in inspector.html, we are trying to load them directly to the webview here.
```

Comment:
nit: wdyt about adding links to the relevant vscode commits that are forcing this change?

Reply:
I think that is reasonable.  Added.